artemis-parent/pom.xml

```diff
@@ -0,0 +1,2074 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+   <modelVersion>4.0.0</modelVersion>
+   <parent>
+      <groupId>org.apache.activemq</groupId>
+      <artifactId>artemis-pom</artifactId>
+      <version>2.33.0-SNAPSHOT</version>
+   </parent>
+
+   <artifactId>artemis-parent</artifactId>

```

Comment:
Something else to consider might be whether we instead introduced a new name at the root pom (e.g artemis-project, artemis-parent...could still use e.g build-parent here later) and leave this as artemis-pom for now, perhaps avoiding the change of effective content breaking anyone...e.g some might be [ab]using it as an import due to lack of specific bom.

(or also like it breaks the now-standalone Examples build, which currently still uses it as a parent as it always had...in part as that was the minimal change during their recent move, but also that there was no bom yet and I didnt want to import the parent during the move or duplicate all the management).